services/userServices.js

```diff
@@ -0,0 +1,57 @@
+const userDao = require('../models/userDao');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const validate = require('../utils/validators');
```

Comment:
순서 조정해주세요~!

- Built-in Module
- Third-party Module
- Custom Module

```suggestion
const jwt = require('jsonwebtoken');
const bcrypt = require('bcrypt');

const userDao = require('../models/userDao');
const validate = require('../utils/validators');
```